godtools/App/Share/Data-DomainInterface/Supporting/GetTranslatedToolCategory/GetTranslatedToolCategory.swift

```diff
@@ -29,8 +29,19 @@ class GetTranslatedToolCategory {
     }
     
     func getTranslatedCategory(resource: ResourceModel, translateInLanguage: BCP47LanguageIdentifier) -> String {
-                
-        let category: String = localizationServices.stringForLocaleElseEnglish(localeIdentifier: translateInLanguage, key: "tool_category_\(resource.attrCategory)")
+        
+        let localeId: String
+        
+        if resource.supportsLanguage(languageId: translateInLanguage) {
+            localeId = translateInLanguage
+        } else {
+            localeId = resource.attrDefaultLocale
```

Comment:
Hey @rachaelblue this looks right for the category to fallback to the default locale.